index.js

```diff
@@ -1,18 +1,85 @@
-'use strict';
+"use strict";
```

Comment:
This files doesn't conform to the `.editorconfig` file.

Reply:
There is no .editorconfig file in this repository. I've formatted it with prettier's default config. Do you want me to change this?

Reply:
Ah, right; and https://github.com/editorconfig/editorconfig/issues/236.

I think what you have is fine, then.